rules-reviewed/eap7/eap71/hibernate51-53.windup.xml

```diff
@@ -224,17 +224,26 @@
                 </hint>
             </perform>
         </rule>
-        <!--https://issues.jboss.org/browse/WINDUPRULE-381
+        <!--https://issues.jboss.org/browse/WINDUPRULE-381 -->
         <rule id="hibernate51-53-00500">
             <when>
+                <javaclass references="org.hibernate.{className}" >
+                    <location>IMPORT</location>
+                </javaclass>
             </when>
             <perform>
+                <hint title="Hibernate 5.3 - RegionFactory usage" effort="3" category-id="mandatory">
+                    <message>Review usage of classes in org.hibernate.cache.spi.RegionFactory, as SPIs for this class has changed.
```

Comment:
Added the ` character to better formatting Java classes' names and 2 white spaces at the end of the row to have the line break to work in the report (otherwise everything will be on the same line)
```suggestion
                    <message>Review usage of classes in `org.hibernate.cache.spi.RegionFactory`, as SPIs for this class has changed.  
```